apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/TimestampBound.cs

```diff
@@ -17,28 +17,53 @@
 namespace Google.Cloud.Spanner.Data
 {
     /// <summary>
+    /// Indicates how Spanner will choose a timestamp at which to read the data for read-only
```

Comment:
(Not for this PR: let's work on making this fully immutable, rather than just externally immutable. Use constructors instead of object initializers etc. Low priority though.)